src/python/doc/installation.rst

```diff
@@ -12,16 +12,16 @@ The easiest way to install the Python version of GUDHI is using
 
 Compiling
 *********
-The library uses c++14 and requires `Boost <https://www.boost.org/>`_ ≥ 1.56.0,
-`CMake <https://www.cmake.org/>`_ ≥ 3.1  to generate makefiles,
+The library uses c++14 and requires `Boost <https://www.boost.org/>`_ :math:`\geq` 1.56.0,
+`CMake <https://www.cmake.org/>`_ :math:`\geq` 3.1  to generate makefiles,
 `NumPy <http://numpy.org>`_, `Cython <https://www.cython.org/>`_ and
 `pybind11 <https://github.com/pybind/pybind11>`_ to compile
 the GUDHI Python module.
 It is a multi-platform library and compiles on Linux, Mac OSX and Visual
 Studio 2017.
 
 On `Windows <https://wiki.python.org/moin/WindowsCompilers>`_ , only Python
-≥ 3.5 are available because of the required Visual Studio version.
+:math:`\geq` 3.5 are available because of the required Visual Studio version.
 
 On other systems, if you have several Python/python installed, the version 2.X
 will be used by default, but you can force it by adding
```

Comment:
Maybe we should specify there that we don't actually support 2.X anymore? (even if because of cmake weirdness we default to it on some platforms...)

Reply:
Ok, but I will have to improve the documentation and maybe making cmake use python3 by default